File: prime.r
a=readline(prompt="enter a number : ")
a=as.integer(a)
flag=0
for(i in 2:as.integer(a/2)){
  if(a%%i==0){
    flag=1
  }
}
if(a<=2){
  flag=0
}
if(flag==0){
  cat(a," is a prime number")
}else{
  cat(a," is not a prime number")
}

File: prime.jl
print("enter a number : ")
n = readline()
n = parse(Int, n)
flag = 0
for i in 2:n/2
    if (n%i == 0)
        global flag = 1
    end
end
if(flag == 1)
    println(n, " is not a prime number")
else 
    println(n, " is a prime number")
end